src/schema/rules/datatypes/derivatives/functional_derivatives.yaml

```diff
@@ -0,0 +1,93 @@
+---
+functional_derivatives_maps:
+  _ref: rules.datatypes.func.func
+  extensions:
+  - ".nii"
+  - ".nii.gz"
```

Comment:
```suggestion
  - _ref: rules.datatypes.func.func.extensions
```

Would this work?

Reply:
I'm not sure if it will work, it doesn't seem to cost much to be explicit. Either way, a thing to note is that it's `$ref`, not `_ref`.